src/user/user.service.ts

```diff
@@ -38,7 +38,10 @@ export class UserService {
   }
 
   // Edit User details
-  async updateUser(UserID, CreateUserDTO: CreateUserDTO): Promise<User> {
+  async updateUser(
+    UserID,
+    CreateUserDTO: CreateUserDTO,
+  ): Promise<User | undefined> {
```

Comment:
Why this change?

Reply:
The promise might return either user type or undefined type, that's why I thought of doing it.